contracts/stellar-axelar-gateway/src/storage.rs

```diff
@@ -1,13 +1,6 @@
 use soroban_sdk::{contracttype, BytesN, String};
 use stellar_axelar_std::contractstorage;
 
-#[contracttype]
-#[derive(Clone, Debug)]
-pub struct MessageApprovalKey {
-    pub source_chain: String,
-    pub message_id: String,
-}
-
 #[contracttype]
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub enum MessageApprovalValue {
```

Comment:
we should create a new value type that doesn't include `NotApproved` variant. This variant is never stored and just used as a default when an approval doesn't exist. We should just use `None` returned by `try_*` getter to make this cleaner